.github/workflows/docker-publish-manual.yaml

```diff
@@ -0,0 +1,69 @@
+name: Docker manual
```

Comment:
Just name it in a different way (filename too) to make it clear it is not the manual workflow of our release docker flow

Reply:
done
thanks!